R/gcloud-config.R

```diff
@@ -17,7 +17,9 @@ gcloud_config <- function(gcloud = NULL) {
     else
       config <- list()
   } else if (is.character(gcloud)) {
-    if (file.exists(gcloud))
+    if (file_test("-d", gcloud) && file_test("-f", file.path(gcloud, "gcloud.yml")))
```

Comment:
Is the idea that `gcloud` can be a directory? I thought it was supposed to be a file? (perhaps we should just discuss realtime)

Reply:
Oh it looks like directory or file. The directory isn't documented and I don't think it's absolutely necessary (but again, let's discuss realtime)

Reply:
Let's try to remove this... Back in the beginning, seems like we used to track the original `CLOUDML_APPLICATION_DIR` with a system environment variable to recover the `cloudml.yml` configuration file; however, with the new configs structure, this seems no longer required since we always persist the `gcloud.yml` file into the current (scope) directory and read this file back as a list.